README.md

```diff
@@ -0,0 +1,176 @@
+
+**Hello Microverse Project** is about mastering all the tools and best practices I've learned about how to document my work in a professional way considering proper Githubflow and especially using other technical topics like linters and pull requests.
+
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">HTML and CSS</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Key Feature1:]**  Code Correction is done by Linter
+- **[Key Feature2:]**  Applied Correct Githubflow and documented in professional way
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- vscode.
+
+- git.
+
+- set up an index.html document.
+- set up a style.css document.
+
+### Setup
+
+Clone this repository to your desired folder:
+- Navigate to the directory where you want clone this project in your computer
+`$ cd Desktop/`
+`$ git clone https://github.com/rahimialijan/Hello-Microverse.git`
+
+### Install
+
+Install this project with this command:
+
+### Usage
+
+To run the project, execute the following command by opening the vscode with the command `code .`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Alijan Rahimi**
+
+- GitHub: [@Alijan-Rahimi](https://github.com/rahimialijan)
+- Twitter: [@AlijanRahimi10](https://twitter.com/AlijanRahimi10)
+- LinkedIn: [Alijan-Rahimi](https://www.linkedin.com/in/alijan-rahimi-18389ab3)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [1] **[ Setting Up A Professional Github Repository.]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+ If you like this project, give it a star ⭐
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for setting up this project for student
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[How is learning in Microverse program? ]**
+
+  - [I have to say , I really enjoy and get a lot of experience.]
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that you add the `MIT` by creating a license file in the project. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose.